### PR TITLE
Copy CLI from Essentials to a new CLI User Guides page

### PR DESCRIPTION
**Pull request by leslie-lau**
Copied the "From the CLI" content from the Essentials pages into a new User Guides section under the CLI dropdown. Note the the existing material on the Essentials pages isn't removed in this PR.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -431,8 +431,19 @@ const config = {
           label: 'Integrations',
         },
         {
+          type: 'dropdown',
           label: 'CLI',
-          to: '/cli/',
+          position: 'left',
```
Comment (cdruxerman) [inline docusaurus.config.js:436]: Instead of setup can you use the term "Reference Docs"

**Review by cdruxerman — COMMENTED** [review]
Added just one comment. I do think we'll need like a cover page of sorts, but it probably doesn't need to block this pr

File: docusaurus.config.js (hunk 1 above)
Comment (leslie-lau) [reply docusaurus.config.js:436]: Updated to "Reference Docs"

**Review by cdruxerman — APPROVED** [review]

**Review by andrewdmaclean — APPROVED** [review]
Looks good to me @leslie-lau !

**Pull request merged**